lib/CleanPlugin.js

```diff
@@ -148,11 +116,12 @@ const getDiffToFs = (fs, outputPath, currentAssets, callback) => {
 				for (const entry of /** @type {string[]} */ (entries)) {
 					const file = entry;
 					const filename = directory ? `${directory}/${file}` : file;
+					const normalizedFilename = path.normalize(filename);
```

Comment:
I think we can cache it, look at `lib/util/identifier.js`, I think we can use the same logic here